rdr_service/genomic/genomic_job_controller.py

```diff
@@ -326,33 +397,73 @@ def run_cvl_reconciliation_report(self):
         except RuntimeError:
             self.job_result = GenomicSubProcessResult.ERROR
 
-    def generate_manifest(self, manifest_type, _genome_type):
+    def generate_manifest(self, manifest_type, _genome_type, **kwargs):
         """
         Creates Genomic manifest using ManifestCompiler component
         """
         self.manifest_compiler = ManifestCompiler(run_id=self.job_run.id,
                                                   bucket_name=self.bucket_name)
         try:
             logging.info(f'Running Manifest Compiler for {manifest_type.name}.')
-            result = self.manifest_compiler.generate_and_transfer_manifest(manifest_type, _genome_type)
-            if result == GenomicSubProcessResult.SUCCESS:
+
+            # Set the feedback manifest name based on the input manifest name
+            if "feedback_record" in kwargs.keys():
+                input_manifest = self.manifest_file_dao.get(kwargs['feedback_record'].inputManifestFileId)
+
+                result = self.manifest_compiler.generate_and_transfer_manifest(manifest_type,
+                                                                               _genome_type,
+                                                                               input_manifest=input_manifest)
+
+            else:
+                result = self.manifest_compiler.generate_and_transfer_manifest(manifest_type, _genome_type)
+
+            if result['code'] == GenomicSubProcessResult.SUCCESS:
                 logging.info(f'Manifest created: {self.manifest_compiler.output_file_name}')
-                # Insert the file record
+
+                new_file_path = f'{self.bucket_name}/{self.manifest_compiler.output_file_name}'
+
+                now_time = clock.CLOCK.now()
```

Comment:
As used here, this is equivalent to datetime.utcnow() (used on line 96), correct?

Reply:
Yes, I'm not sure why I'm doing it two different ways. I can update them both to use datetime.utcnow()